astropy/io/fits/convenience.py

```diff
@@ -443,7 +443,8 @@ def table_to_hdu(table, character_as_bytes=False):
         The FITS binary table HDU.
     """
     # Avoid circular imports
-    from .connect import is_column_keyword, REMOVE_KEYWORDS
+    from .connect import is_column_keyword, REMOVE_KEYWORDS, python_to_tdisp
+    from ...table.column import BaseColumn
```

Comment:
Are you sure this is OK? I ask because of thee comment in l.454

Reply:
I have no idea, I hadn't noticed that comment.  I know it works for me test runs.  What is erfa?

Reply:
Irrelevant now...